src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -342,15 +343,35 @@ public Response setMetadataRoot( @PathParam("identifier")String dvIdtf, String b
     
     @GET
     @Path("{identifier}/facets/")
+    /**
+     * return list of facets for the dataverse with alias `dvIdtf`
+     */
     public Response listFacets( @PathParam("identifier") String dvIdtf ) {
```

Comment:
It would be nice to add at least one test to https://github.com/IQSS/dataverse/blob/v4.6/src/test/java/edu/harvard/iq/dataverse/api/DataversesIT.java to make sure this API doesn't stop working again. (I assume it worked once upon a time.)